tests/test_physics.py

```diff
@@ -150,8 +152,9 @@ def test_dynamics(sim):
 
             # TODO: expose object properties (such as mass) to python
             # Counter the force of gravity on the object (it should not translate)
-            obj_mass = 0.41
-            sim.apply_force(np.array([0, obj_mass * 9.8, 0]), np.zeros(3), object_id)
+            sim.apply_force(
+                np.array([0, object1_mass * 9.8, 0]), np.zeros(3), object_id
```

Comment:
'G' (9.8) should really be defined as a constant somewhere, shouldn't it?

Reply:
Thanks, some of this was written with early bindings.